docs/structure-of-a-contract.rst

```diff
@@ -42,6 +42,41 @@ Functions are the executable units of code within a contract.
 and have different levels of visibility (:ref:`visibility-and-getters`)
 towards other contracts. Functions must be decorated with either @public or @private.
 
+Default function
+----------------
+
+A contract can also have a default function, which is executed on a call to the contract if no other functions match the given function identifier (or if none was supplied at all, such as through someone sending it Eth). It is the same construct as fallback functions `in Solidity <https://solidity.readthedocs.io/en/latest/contracts.html?highlight=fallback#fallback-function>`_. 
+
+This function is always named `__default__` and must be annotated with `@public` and `@payable`. It cannot have arguments and cannot return anything.
+
+Furthermore, this function is executed whenever the contract receives plain Ether (without data). This is why the default function cannot accept arguments and return values - it is a design decision of Ethereum to make no differentiation between sending ether to a contract or a user address.
+
+**Example:**
+
+::
+
+    Sent: event({sender: indexed(address)})
```

Comment:
Suggestion: maybe add `amount` to this event, so if someone tries to run it you can see the amount you sent in the event.

Reply:
Good idea.